source/2024.tex

```diff
@@ -0,0 +1,101 @@
+\end{tabular}
+
+\section{Objectives and Activities}
+The objectives of the UKPA are:
+\begin{enumerate}
+	\item To advance education for the public benefit in the use and understanding of the Python programming language within the UK by:
+	      \begin{enumerate}
+		      \item Organising, presenting and promoting Python related conferences, meetings and events at venues within the UK,
+		      \item Supporting and promoting Python related projects and events organised by UKPA members, and,
+		      \item Supporting UKPA members wishing to attend Python related conferences, meetings and events.
+	      \end{enumerate}
+	\item To advance education for the public benefit in the use and understanding of the Python programming language worldwide by supporting and promoting overseas Python related conferences, meetings and events.
+\end{enumerate}
+
+The primary activity of the UKPA is to organise and run the annual PyCon UK conference.
+
+\section{Structure, Governance and Management}
+
+\subsection{Governing Document}
+The UKPA's governing document is its constitution, the current version of which is available at https://github.com/PyconUK/ukpa-constitution/releases/latest.
+
+\subsection{UKPA Members}
+The UKPA membership is the electoral body to which the trustees are accountable.
+There are two classes of membership:
+
+\begin{enumerate}
+	\item Members: have the full rights and obligations described in the UKPA constitution, including the right to vote on members decisions.
+	\item Associate Members: non-voting class of membership as described in the UKPA constitution.
+\end{enumerate}
+
+\subsection{Trustee Appointment}
+At the AGM held on 24-Sep-2023, Vince Knight retired as a trustee in accordance with section 13.1.1 of the UKPA's constitution.
+
+Becky Smith also retired as a trustee in accordance with sections 13.1.4 and 13.1.5 of the constitution.
+
+Vincent Knight and Becky Smith were elected as trustees for a further term in accordance with section 13.1.4 of the constitution.
+
+On 04-Oct-2023, Vince Knight resigned as a trustee.
+
+By a resolution dated 10-Oct-2023, Daniele Procida was appointed as a trustee of the UKPA in accordance with clause 13.1.4 of the constitution
+\subsection{Risk and Internal Control}
+The trustees have identified the major risks to which they believe the UKPA is exposed. These are:
+
+\subsubsection{Reduction or Loss of Members}
+If there were a reduction in membership of the UKPA then there would have to be a contraction, consolidation or closure of its activities.
+
+
+\subsubsection{Reduction or Loss of Volunteers}
+The UKPA is totally reliant upon volunteers to run and administer the activities of the UKPA. If there were a reduction in the number of volunteers to an unacceptable level, then there would have to be a contraction, consolidation or closure of activities.
+
+\section{Achievements and Performance}
+PyCon UK 2023 was held at Cardiff City Hall from 22nd to 25th September 2023.
```

Comment:
Worth mentioning that we provided a grant to the Python in Education day?

Reply:
No, because that's outside the reporting date range - but DCEU wasn't and neither was DjangoCon Africa!